docs/deployments/overview.md

```diff
@@ -0,0 +1,58 @@
+# Overview
+
+## Introduction
+
+To ensure the integrity and security of contracts uploaded to the network, it is necessary to verify the SHA256 checksums of both the local binary file of the contract and the contract deployed on the network. This document outlines the steps for performing this verification.
+
+## Tools
```

Comment:
Please add
1. how to find proper version of the contract
2. how to build contracts

Reply:
what do you mean by contract version? Git commit that was used to build binaries? Information about it in the "Contracts and Their Checksums" section, but I can reorganize information